traveler-ios-ui/TravelerKitUI/ViewControllers/CatalogItemResultViewController.swift

```diff
@@ -22,6 +23,7 @@ class CatalogItemResultViewController: UIViewController {
     @IBOutlet weak var descriptionLabel: UILabel!
     @IBOutlet weak var itemInfoHeightConstraint: NSLayoutConstraint!
     @IBOutlet weak var itemInfoView: UIView!
+    @IBOutlet weak var wishlistButton: HeartButton!
```

Comment:
If you piggy back off of `isSelected` then this becomes a regular UIButton and the VC doesnt need to know what kind of button it is.